ConnectivityTestScriptsForWindows/Publish-Report.ps1

```diff
@@ -65,8 +78,16 @@ $AzureBackupCommonIssue =
 <ul>
 <li> <b>Azure Workload backup doesn't support authenticated Proxy servers. <a href ='https://docs.microsoft.com/azure/backup/backup-sql-server-database-azure-vms#use-an-http-proxy-server-to-route-traffic'> Use an HTTP proxy server to route traffic</a> </b></li>
 <li> Azure Workload Backup only support TLS 1.2 . Change TLS registry settings as described <a href = 'https://docs.microsoft.com/windows-server/security/tls/tls-registry-settings'>here</a> to enable TLS 1.2 .</li>
-<li> If not using Private Endpoint, check Network Security Group (NSG) attached to subnet, NSG should allow outbound connection to Service Tag AzureBackup <a href = 'https://docs.microsoft.com/azure/virtual-network/service-tags-overview'>Service Tags. </a>
-Check firewall settings for outbound connection to URLs mentioned in <a href='https://docs.microsoft.com/azure/backup/backup-sql-server-database-azure-vms#allow-access-to-service-fqdns'> this section </a></li>
+<li> If not using Private Endpoint, check Network Security Group (NSG) attached to subnet, NSG should allow outbound connection to AzureBackup <a href = 'https://docs.microsoft.com/azure/virtual-network/service-tags-overview'>Service Tags. </a>
```

Comment:
If the Azure Recovery Services vault is not using Private Endpoint....

Reply:
to the subnet

Reply:
check the Network Security...

Reply:
Done